exercises/two-bucket/src/example.h

```diff
@@ -0,0 +1,24 @@
+#ifndef TWO_BUCKET_H
+#define TWO_BUCKET_H
+
+#include <stdbool.h>
+
+typedef enum {
+   bucket_id_1,
```

Comment:
nit: I believe we usually use ALL CAPS enums.

Reply:
Yep, you're right. I'll get this fixed.